recipes/openssl/3.x.x/conanfile.py

```diff
@@ -386,7 +384,8 @@ def _configure_args(self):
 
         args.append("no-fips" if self.options.get_safe("no_fips", True) else "enable-fips")
         args.append("no-md2" if self.options.get_safe("no_md2", True) else "enable-md2")
-        args.append("-DOPENSSL_TLS_SECURITY_LEVEL=%s" % str(self.options.tls_security_level))
+        if str(self.options.tls_security_level) != "None":
```

Comment:
Both None and 0 are negative boolean values. Plus, `self.options.tls_security_level` is a Conan object, it's converted according to the operation needed.